python/pynessie/model.py

```diff
@@ -169,7 +169,7 @@ def to_string(self) -> str:
         """Convert this key to friendly CLI string."""
         # false positives in pylint
         # pylint: disable=E1133
-        return ".".join(f'"{i}"' if "." in i else i for i in self.elements)
+        return ".".join(f'"{i!s}"' if "." in i else i for i in self.elements)  # noqa: B028
```

Comment:
is it intentional to change both the format string as well as add the suppression?
afaict only one of the 2 would be needed